dbt/utils.py

```diff
@@ -39,11 +40,14 @@ def __init__(self, profile, adapter, node, use_temp=False):
         self.node = node
         self.schema = node.get('schema')
         self.name = node.get('name')
+        # set alias, defaults to name
+        self.alias = get_alias(node)
+        self.node['alias'] = get_alias(node)
```

Comment:
oh! Let's not mutate the `node` here!

I think `node['alias']` should be set over [here](https://github.com/fishtown-analytics/dbt/blob/development/dbt/parser.py#L222) instead. The `schema` attribute is set on line 222 and 240 in that file. I think we'll want to do something similar here for `alias`.

This should also fix the issue with the `START` line log output I believe

Reply:
not mutating the node there caused my initial tests to fail